.github/workflows/generate-docs-for-dbt-core.yml

```diff
@@ -171,40 +182,47 @@ jobs:
       - name: Checkout dbt-docs repo
         uses: actions/checkout@v3
         with:
-          path: main
+          repository: dbt-labs/dbt-docs # redundant but calling it out for clarity
       
+      # if there are no changie files to delete the job fails because the PR has 0
+      # changes, so catch it here for clarity.
+      - name: Check for changed changie yaml files
+        id: changie_check
+        run: |
+          if [[ -z $(find ./.changes/unreleased -type f -name "*.yaml") ]] then
+            echo No changie yaml files to delete.  This is unexpected.
+            exit 1
+          fi
+
       - name: Create branch
         run: |
-          git checkout -b ${{needs.set_variables.outputs.branch_name}}
-          git push -u origin ${{needs.set_variables.outputs.branch_name}}
+          git checkout -b ${{ needs.set_variables.outputs.branch_name }}
+          git push -u origin ${{ needs.set_variables.outputs.branch_name }}
       
       - name: Check current branch
         run: |
           git status
-      
+
       - name: Remove changie yaml files
         id: rm_yml
         run: |
-          find . -name '.changes/unreleased/*.yml' -delete
-
-      - name: Print branch status
-        run: |
-          git status
+          find ./.changes/unreleased -type f -name "*.yaml" -delete
 
       - name: Commit & Push changes
         run: |
           git config user.name 'Github Build Bot'
           git config user.email 'buildbot@fishtownanalytics.com'
           git pull
-          git add .
+          git add .changes/unreleased
           git commit -m "Delete changie files that were moved to dbt-core for release"
           git push
 
       - name: Create Pull Request
-        run:
+        run: |
           gh pr create --title "Remove moved changie yml files" \
-                       --body ${{ needs.set_variables.outputs.pr_body }} \
+                       --body "${{ needs.set_variables.outputs.pr_body }}" \
                        --base main \
-                       --head ${{ needs.set_variables.outputs.branch_name }
+                       --head ${{ needs.set_variables.outputs.branch_name }}
+                       --label "Skip Changelog"
         env:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Should this be `${{ secrets.FISHTOWN_BOT_PAT }}` ?

Reply:
No.  Since this is accessing the `dbt-docs` repo, it can use the token generated by the action.  The token will have the permissions from the `permission` block above.   Above we need to use the PAT because it's trying to commit and open a PR against `dbt-cpre`, a repo outside the context (`dbt-docs` repo) that the action is running in.

Reply:
Thanks for explaining!!!